### PR TITLE
chore: fix cifar runtime

### PR DESCRIPTION
**Pull request by andrei-stoian-zama**
Closes https://github.com/zama-ai/concrete-ml-internal/issues/4047

### CODE REVIEW

**Review by RomanBredehoft — APPROVED** [review]

File: use_case_examples/cifar/cifar_brevitas_training/README.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ Experiments were conducted on an m6i.metal machine offering 128 CPU cores and 51
 | VGG FHE (simulation\*) | 6 bits   | 86.0     |
 | VGG FHE                | 6 bits   | 86.0\*\* |
 
-We ran the FHE inference over 10 examples and achieved 100% similar predictions between the simulation and FHE. The overall accuracy for the entire data-set is expected to match the simulation. The original model with a maximum of 13 bits of precision ran in around 9 hours on the specified hardware. Using the rounding approach, the final model ran in **31 minutes**, providing a speedup factor of 18x while preserving accuracy. This significant performance improvement demonstrates the benefits of the rounding operator in the FHE setting.
+We ran the FHE inference over 10 examples and achieved 100% similar predictions between the simulation and FHE. The overall accuracy for the entire data-set is expected to match the simulation. The original model (no rounding) with a maximum of 13 bits of precision runs in around 9 hours on the specified hardware. Using the rounding approach, the final model ran in **4 minutes**. This significant performance improvement demonstrates the benefits of the rounding operator in the FHE setting.
```
Comment (jfrery) [inline use_case_examples/cifar/cifar_brevitas_training/README.md:101]: I think it's worth tracking the performance improvement. Something like: jan 2023: 31min, august 2023: 4min.

**Review by fd0r — APPROVED** [review]

File: use_case_examples/cifar/cifar_brevitas_training/README.md (hunk 1 above)
Comment (fd0r) [reply use_case_examples/cifar/cifar_brevitas_training/README.md:101]: Not sure if useful for the user

Comment (RomanBredehoft) [reply use_case_examples/cifar/cifar_brevitas_training/README.md:101]: they are supposed to be up to date for the past 3 months at most, so I also don't think it's very relevant !

Comment (jfrery) [reply use_case_examples/cifar/cifar_brevitas_training/README.md:101]: I would be reassured to see that there was a 30x speed up in the past 6 months. But I heard the masses. Let's not do it.

**Review by jfrery — APPROVED** [review]

**Pull request merged**